### ISSUE TITLE
[Bug] ode_causal_pipeline.py: error: unrecognized arguments: --override-transformer-cls-name CausalWanTransformer3DModel

### ISSUE THREAD

**Issue by lihuining** [hao-ai-lab/FastVideo#835]
### Describe the bug

bash /root/paddlejob/workspace/env_run/cuda_compact/FastVideo/examples/training/consistency_finetune/causal_ode_init/finetune_ode_init.sh
INFO 10-09 20:50:44 [__init__.py:86] ROCm platform is unavailable: No module named 'amdsmi'
WARNING 10-09 20:50:44 [logger.py:122]  By default, logger.info(..) will only log from the local main process. Set logger.info(..., is_local_main_process=False) to log from all processes.
INFO 10-09 20:50:44 [__init__.py:47] CUDA is available
usage: ode_causal_pipeline.py [-h] --data-path DATA_PATH --dataloader-num-workers
                              DATALOADER_NUM_WORKERS --num-height NUM_HEIGHT
                              --num-width NUM_WIDTH --num-frames NUM_FRAMES
                              --train-batch-size TRAIN_BATCH_SIZE --num-latent-t
                              NUM_LATENT_T [--group-frame [GROUP_FRAME]]
                              [--group-resolution [GROUP_RESOLUTION]]
                              --pretrained-model-name-or-path
                              PRETRAINED_MODEL_NAME_OR_PATH
                              [--dit-model-name-or-path DIT_MODEL_NAME_OR_PATH]
                              [--cache-dir CACHE_DIR]
                              [--generator-model-path GENERATOR_MODEL_PATH]
                              [--real-score-model-path REAL_SCORE_MODEL_PATH]
                              [--fake-score-model-path FAKE_SCORE_MODEL_PATH]
                              [--ema-decay EMA_DECAY]
                              [--ema-start-step EMA_START_STEP]
                              [--training-cfg-rate TRAINING_CFG_RATE]
                              [--precondition-outputs [PRECONDITION_OUTPUTS]]
                              [--validation-dataset-file VALIDATION_DATASET_FILE]
                              [--validation-preprocessed-path VALIDATION_PREPROCESSED_PATH]
                              [--validation-sampling-steps VALIDATION_SAMPLING_STEPS]
                              [--validation-guidance-scale VALIDATION_GUIDANCE_SCALE]
                              [--validation-steps VALIDATION_STEPS]
                              [--log-validation [LOG_VALIDATION]]
                              [--tracker-project-name TRACKER_PROJECT_NAME]
                              [--wandb-run-name WANDB_RUN_NAME] [--seed SEED]
                              --output-dir OUTPUT_DIR
                              [--checkpoints-total-limit CHECKPOINTS_TOTAL_LIMIT]
                              [--checkpointing-steps CHECKPOINTING_STEPS]
                              [--training-state-checkpointing-steps TRAINING_STATE_CHECKPOINTING_STEPS]
                              [--weight-only-checkpointing-steps WEIGHT_ONLY_CHECKPOINTING_STEPS]
                              [--resume-from-checkpoint RESUME_FROM_CHECKPOINT]
                              [--init-weights-from-safetensors INIT_WEIGHTS_FROM_SAFETENSORS]
                              [--logging-dir LOGGING_DIR]
                              [--num-train-epochs NUM_TRAIN_EPOCHS]
                              [--max-train-steps MAX_TRAIN_STEPS]
                              [--gradient-accumulation-steps GRADIENT_ACCUMULATION_STEPS]
                              --learning-rate LEARNING_RATE
                              [--scale-lr [SCALE_LR]]
                              [--lr-scheduler LR_SCHEDULER]
                              [--lr-warmup-steps LR_WARMUP_STEPS]
                              [--max-grad-norm MAX_GRAD_NORM]
                              [--enable-gradient-checkpointing-type {full,ops,block_skip}]
                              [--selective-checkpointing SELECTIVE_CHECKPOINTING]
                              [--mixed-precision MIXED_PRECISION]
                              [--train-sp-batch-size TRAIN_SP_BATCH_SIZE]
                              [--fsdp-sharding-strategy FSDP_SHARDING_STRATEGY]
                              [--weighting_scheme {sigma_sqrt,logit_normal,mode,cosmap,uniform}]
                              [--logit_mean LOGIT_MEAN] [--logit_std LOGIT_STD]
                              [--mode_scale MODE_SCALE]
                              [--num-euler-timesteps NUM_EULER_TIMESTEPS]
                              [--lr-num-cycles LR_NUM_CYCLES]
                              [--lr-power LR_POWER] [--min-lr-ratio MIN_LR_RATIO]
                              [--not-apply-cfg-solver [NOT_APPLY_CFG_SOLVER]]
                              [--distill-cfg DISTILL_CFG]
                              [--scheduler-type SCHEDULER_TYPE]
                              [--linear-quadratic-threshold LINEAR_QUADRATIC_THRESHOLD]
                              [--linear-range LINEAR_RANGE]
                              [--weight-decay WEIGHT_DECAY] [--betas BETAS]
                              [--use-ema [USE_EMA]]
                              [--multi-phased-distill-schedule MULTI_PHASED_DISTILL_SCHEDULE]
                              [--pred-decay-weight PRED_DECAY_WEIGHT]
                              [--pred-decay-type PRED_DECAY_TYPE]
                              [--hunyuan-teacher-disable-cfg [HUNYUAN_TEACHER_DISABLE_CFG]]
                              [--master-weight-type MASTER_WEIGHT_TYPE]
                              [--VSA-decay-rate VSA_DECAY_RATE]
                              [--VSA-decay-interval-steps VSA_DECAY_INTERVAL_STEPS]
                              [--lora-training [LORA_TRAINING]]
                              [--lora-rank LORA_RANK] [--lora-alpha LORA_ALPHA]
                              [--moba-config-path MOBA_CONFIG_PATH]
                              [--generator-update-interval GENERATOR_UPDATE_INTERVAL]
                              [--dfake-gen-update-ratio DFAKE_GEN_UPDATE_RATIO]
                              [--min-timestep-ratio MIN_TIMESTEP_RATIO]
                              [--max-timestep-ratio MAX_TIMESTEP_RATIO]
                              [--real-score-guidance-scale REAL_SCORE_GUIDANCE_SCALE]
                              [--fake-score-learning-rate FAKE_SCORE_LEARNING_RATE]
                              [--fake-score-betas FAKE_SCORE_BETAS]
                              [--fake-score-lr-scheduler FAKE_SCORE_LR_SCHEDULER]
                              [--log-visualization [LOG_VISUALIZATION]]
                              [--simulate-generator-forward [SIMULATE_GENERATOR_FORWARD]]
                              [--warp-denoising-step [WARP_DENOISING_STEP]]
                              [--num-frame-per-block NUM_FRAME_PER_BLOCK]
                              [--independent-first-frame [INDEPENDENT_FIRST_FRAME]]
                              [--enable-gradient-masking [ENABLE_GRADIENT_MASKING]]
                              [--gradient-mask-last-n-frames GRADIENT_MASK_LAST_N_FRAMES]
                              [--validate-cache-structure [VALIDATE_CACHE_STRUCTURE]]
                              [--same-step-across-blocks [SAME_STEP_ACROSS_BLOCKS]]
                              [--last-step-only [LAST_STEP_ONLY]]
                              [--context-noise CONTEXT_NOISE]
                              [--model-path MODEL_PATH] [--model-dir MODEL_DIR]
                              [--mode {inference,preprocess,finetuning,distillation}]
                              [--workload-type {i2v,t2v,t2i,i2i}]
                              [--distributed-executor-backend {mp}]
                              [--inference-mode [INFERENCE_MODE]]
                              [--trust-remote-code [TRUST_REMOTE_CODE]]
                              [--revision REVISION] [--num-gpus NUM_GPUS]
                              [--tp-size TP_SIZE] [--sp-size SP_SIZE]
                              [--hsdp-replicate-dim HSDP_REPLICATE_DIM]
                              [--hsdp-shard-dim HSDP_SHARD_DIM]
                              [--dist-timeout DIST_TIMEOUT] [--output-type {pil}]
                              [--prompt-txt PROMPT_TXT]
                              [--STA-mode {STA_inference,STA_searching,STA_tuning,STA_tuning_cfg,None}]
                              [--skip-time-steps SKIP_TIME_STEPS]
                              [--mask-strategy-file-path MASK_STRATEGY_FILE_PATH]
                              [--enable-torch-compile [ENABLE_TORCH_COMPILE]]
                              [--dit-cpu-offload [DIT_CPU_OFFLOAD]]
                              [--use-fsdp-inference [USE_FSDP_INFERENCE]]
                              [--text-encoder-cpu-offload [TEXT_ENCODER_CPU_OFFLOAD]]
                              [--image-encoder-cpu-offload [IMAGE_ENCODER_CPU_OFFLOAD]]
                              [--vae-cpu-offload [VAE_CPU_OFFLOAD]]
                              [--pin-cpu-memory [PIN_CPU_MEMORY]]
                              [--disable-autocast [DISABLE_AUTOCAST]]
                              [--VSA-sparsity VSA_SPARSITY]
                              [--master-port MASTER_PORT]
                              [--enable-stage-verification [ENABLE_STAGE_VERIFICATION]]
                              [--pipeline-config-path PIPELINE_CONFIG_PATH]
                              [--embedded-cfg-scale EMBEDDED_CFG_SCALE]
                              [--flow-shift FLOW_SHIFT]
                              [--dit-precision {fp32,fp16,bf16}]
                              [--vae-precision {fp32,fp16,bf16}]
                              [--vae-tiling [VAE_TILING]] [--vae-sp [VAE_SP]]
                              [--text-encoder-precisions {fp32,fp16,bf16} [{fp32,fp16,bf16} ...]]
                              [--image-encoder-precision {fp32,fp16,bf16}]
                              [--pos_magic POS_MAGIC] [--neg_magic NEG_MAGIC]
                              [--timesteps_scale TIMESTEPS_SCALE]
                              [--dmd-denoising-steps DMD_DENOISING_STEPS]
                              [--vae-config.load-encoder [VAE_CONFIG.LOAD_ENCODER]]
                              [--vae-config.load-decoder [VAE_CONFIG.LOAD_DECODER]]
                              [--vae-config.tile-sample-min-height VAE_CONFIG.TILE_SAMPLE_MIN_HEIGHT]
                              [--vae-config.tile-sample-min-width VAE_CONFIG.TILE_SAMPLE_MIN_WIDTH]
                              [--vae-config.tile-sample-min-num-frames VAE_CONFIG.TILE_SAMPLE_MIN_NUM_FRAMES]
                              [--vae-config.tile-sample-stride-height VAE_CONFIG.TILE_SAMPLE_STRIDE_HEIGHT]
                              [--vae-config.tile-sample-stride-width VAE_CONFIG.TILE_SAMPLE_STRIDE_WIDTH]
                              [--vae-config.tile-sample-stride-num-frames VAE_CONFIG.TILE_SAMPLE_STRIDE_NUM_FRAMES]
                              [--vae-config.blend-num-frames VAE_CONFIG.BLEND_NUM_FRAMES]
                              [--vae-config.use-tiling [VAE_CONFIG.USE_TILING]]
                              [--vae-config.use-temporal-tiling [VAE_CONFIG.USE_TEMPORAL_TILING]]
                              [--vae-config.use-parallel-tiling [VAE_CONFIG.USE_PARALLEL_TILING]]
                              [--dit-config.prefix DIT_CONFIG.PREFIX]
                              [--dit-config.quant-config DIT_CONFIG.QUANT_CONFIG]
                              [--preprocess.model-path PREPROCESS.MODEL_PATH]
                              [--preprocess.dataset-path PREPROCESS.DATASET_PATH]
                              [--preprocess.dataset-type {hf,merged}]
                              [--preprocess.dataset-output-dir PREPROCESS.DATASET_OUTPUT_DIR]
                              [--preprocess.dataloader-num-workers PREPROCESS.DATALOADER_NUM_WORKERS]
                              [--preprocess.preprocess-video-batch-size PREPROCESS.PREPROCESS_VIDEO_BATCH_SIZE]
                              [--preprocess.samples-per-file PREPROCESS.SAMPLES_PER_FILE]
                              [--preprocess.flush-frequency PREPROCESS.FLUSH_FREQUENCY]
                              [--preprocess.video-loader-type {torchcodec,torchvision}]
                              [--preprocess.max-height PREPROCESS.MAX_HEIGHT]
                              [--preprocess.max-width PREPROCESS.MAX_WIDTH]
                              [--preprocess.num-frames PREPROCESS.NUM_FRAMES]
                              [--preprocess.video-length-tolerance-range PREPROCESS.VIDEO_LENGTH_TOLERANCE_RANGE]
                              [--preprocess.train-fps PREPROCESS.TRAIN_FPS]
                              [--preprocess.speed-factor PREPROCESS.SPEED_FACTOR]
                              [--preprocess.drop-short-ratio PREPROCESS.DROP_SHORT_RATIO]
                              [--preprocess.do-temporal-sample [PREPROCESS.DO_TEMPORAL_SAMPLE]]
                              [--preprocess.training-cfg-rate PREPROCESS.TRAINING_CFG_RATE]
                              [--preprocess.seed PREPROCESS.SEED]
ode_causal_pipeline.py: error: unrecognized arguments: --override-transformer-cls-name CausalWanTransformer3DModel

### Reproduction

bash FastVideo/examples/training/consistency_finetune/causal_ode_init/finetune_ode_init.sh

### Environment

torch 2.7.1
cuda 12.4

**Comment by zhisbug** [comment]
@JerryZhou54 please take a look and answer

**Comment by JerryZhou54** [comment]
Ode init should be fully runnable without bugs after this PR is merged: https://github.com/hao-ai-lab/FastVideo/pull/1084
